packages/sns/src/sns.wrapper.ts

```diff
@@ -126,6 +141,100 @@ export class SnsWrapper {
     params: Omit<SnsGetNeuronParams, "certified">
   ): Promise<Neuron> => this.governance.getNeuron(this.mergeParams(params));
 
+  /**
+   * Returns the subaccount of the next neuron to be created.
+   *
+   * The neuron account is a subaccount of the governance canister.
+   * The subaccount is derived from the controller and an ascending index.
+   *
+   * @param controller
+   * @returns
+   */
+  getNextNeuronAccount = async (
+    controller: Principal
+  ): Promise<{ account: SnsAccount; index: bigint }> => {
+    for (let index = 0; index < MAX_NEURONS_SUBACCOUNTS; index++) {
+      const subaccount = this.getNeuronSubaccount({ index, controller });
+      const account = {
+        owner: this.canisterIds.governanceCanisterId,
+        subaccount,
+      };
+      let balance = await this.ledger.balance({ ...account, certified: false });
+      if (balance === BigInt(0)) {
+        // Recheck the balance with an update call
+        balance = await this.ledger.balance({ ...account, certified: true });
+        // If the balance is still 0, we can use this subaccount
+        if (balance === BigInt(0)) {
+          return {
+            account,
+            index: BigInt(index),
+          };
+        }
+      }
+    }
+    throw new SnsGovernanceError("No more neuron accounts available");
+  };
+
+  /**
+   * Stakes a neuron.
+   *
+   * This is a convenient method that transfers the stake to the neuron subaccount and then claims the neuron.
+   *
+   * @param {SnsStakeNeuronParams} params
+   * @returns
+   */
+  stakeNeuron = async ({
+    stakeE8s,
+    source,
+    controller,
+  }: SnsStakeNeuronParams): Promise<NeuronId> => {
+    if (!this.certified) {
```

Comment:
if we don't have it yet not for this PR but maybe we can have a generic `assertCertified` function?

Reply:
Sure.